src/raster/r.hydro.flatten/r.hydro.flatten.html

```diff
@@ -79,4 +79,4 @@ <h2>SEE ALSO</h2>
 
 <h2>AUTHOR</h2>
 
-Anna Petrasova, <a href="http://geospatial.ncsu.edu/geoforall/">NCSU GeoForAll Lab</a>
+Anna Petrasova, <a href="https://geospatial.ncsu.edugeoforall/">NCSU GeoForAll Lab</a>
```

Comment:
Both "osgeorel" and "geoforall" variations are affected by the missing slash.

Reply:
Ouch. Fixed in 4a4fcd044318e0211cace904208dd4f89dc870c1